test/spec/MfaVerify_spec.js

```diff
@@ -346,7 +338,6 @@ function (Q, _, $, Duo, OktaAuth, LoginUtil, CryptoUtil, CookieUtil, Util, MfaVe
           when a "show answer" checkbox is checked', function () {
           return setupSecurityQuestion().then(function (test) {
             var answer = test.form.answerField();
```

Comment:
I would expect to see some extra testing around the functionality you just added - for example, that we have the correct button states as we're toggling on/off (the show button is present first, when you click the button it's now the hide button).

In general, if you're adding functionality, you should be adding tests.